src/list.tsx

```diff
@@ -643,12 +646,12 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
     return (
         <div
             className={classNames(styles.rowContent, styles.rowContentPadding)}
-            role={role}
+            role={touchableRole ?? role}
```

Comment:
why using touchableRole here? this isn't a touchable

Reply:
you totally right, Me Se fue la pinza XD